src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -4271,29 +4273,29 @@ private static void DescribeExpression(TextWriter writer, RegexNode node, string
 
                 // Write out the line for the node.
                 const char BulletPoint = '\u25CB';
-                writer.WriteLine($"{prefix}{new string(' ', depth * 4)}{BulletPoint} {tag}{DescribeNode(node, analysis)}");
+                writer.WriteLine($"/// {new string(' ', depth * 4)}{BulletPoint} {tag}{HttpUtility.HtmlEncode(DescribeNode(node, rm))}<br/>");
```

Comment:
I suppose HttpUtility.HtmlEncode will handle cases where someone has <br/> or </code> or </remarks> in their pattern for some reason?

Reply:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/2642209/163276442-cd9ae377-9dd1-4be3-ba72-d3581eeb6d7a.png">